ehitchhiking-ui/src/app/pages/registration/registration.component.html

```diff
@@ -0,0 +1,33 @@
+<div class = "page">
+<div id="login-form">
+    <div class = "exadel-login">
+      <img src="../../../assets/images/FullColor_Word-1.png" alt="exadel-logo"/>
```

Comment:
pls rename image (for example, exadel-logo.png)

Reply:
rename to exadel-logo-horizon